CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 - [#4790](https://github.com/influxdb/influxdb/pull/4790): Allow openTSDB point-level error logging to be disabled
+- [#4728](https://github.com/influxdb/influxdb/pull/4728): SHOW SHARD GROUPS
```

Comment:
Please add 'by @mateuszdyminski' to the end of the line. We like to include credits in the CHANGELOG.